packages/calcite-components/src/components/shell-panel/shell-panel.e2e.ts

```diff
@@ -118,7 +135,7 @@ describe("calcite-shell-panel", () => {
       "calcite-shell-panel",
       (panel: HTMLCalciteShellPanelElement, containerClass: string, contentClass: string) => {
         const container = panel.shadowRoot.querySelector(containerClass);
-        return container.firstElementChild.className == contentClass;
+        return container.firstElementChild.className.split(" ")[0] == contentClass;
```

Comment:
Can you use `container.firstElementChild.classList.contains(contentClass)` instead? The current approach is brittle.